extensions/tika/runtime/src/main/java/io/quarkus/tika/runtime/TikaRecorder.java

```diff
@@ -47,9 +47,10 @@ private static InputStream getTikaConfigStream(TikaConfiguration config, String
         // Load tika-config.xml resource
         InputStream is;
         if (config.tikaConfigPath.isPresent()) {
-            is = TikaRecorder.class.getResourceAsStream(
-                    config.tikaConfigPath.get().startsWith("/") ? config.tikaConfigPath.get()
-                            : "/" + config.tikaConfigPath.get());
+            is = TikaRecorder.class.getResourceAsStream(config.tikaConfigPath.get());
+            if (is == null) {
+            	is = Thread.currentThread().getContextClassLoader().getResourceAsStream(config.tikaConfigPath.get());
```

Comment:
I think you should use the TCCL directly here.

Reply:
:+1:

Reply:
Done